src/sonic-config-engine/tests/sample_output/py3/mvrf_interfaces

```diff
@@ -34,14 +34,14 @@ iface eth0 inet static
     up ip -4 route add default via 10.0.0.1 dev eth0 table 5000 metric 201
     up ip -4 route add 10.0.0.0/24 dev eth0 table 5000
     up ip -4 rule add pref 32765 from 10.0.0.100/32 table 5000
-    up ip rule add pref 32764 to 11.11.11.11 table 5000
-    up ip rule add pref 32764 to 22.22.22.0/23 table 5000
+    up ip -4 rule add pref 32764 to 11.11.11.11 table 5000
+    up ip -4 rule add pref 32764 to 22.22.22.0/23 table 5000
     # management port down rules
     pre-down ip -4 route delete default via 10.0.0.1 dev eth0 table 5000
     pre-down ip -4 route delete 10.0.0.0/24 dev eth0 table 5000
     pre-down ip -4 rule delete pref 32765 from 10.0.0.100/32 table 5000
-    pre-down ip rule delete pref 32764 to 11.11.11.11 table 5000
-    pre-down ip rule delete pref 32764 to 22.22.22.0/23 table 5000
+    pre-down ip -4 rule delete pref 32764 to 11.11.11.11 table 5000
+    pre-down ip -4 rule delete pref 32764 to 22.22.22.0/23 table 5000
```

Comment:
the title is about ipv4, but new test cases are all ipv4, am i missing something here?

Reply:
I will add new test case for IPV6.

For IPV4 address, following 2 commands are exactly same:
up ip rule add pref 32764 to 11.11.11.11 table 5000
up ip **-4** rule add pref 32764 to 11.11.11.11 table 5000


But for IPV6 address, we always need add '-6' parameter:
up ip **-6** rule add pref 32764 to 2603:10e2:0:2902::8 table 5000

Reply:
Fixed.